F/FastJet_Julia_Wrapper/build_tarballs.jl

```diff
@@ -3,10 +3,9 @@
 using BinaryBuilder
 using Pkg
 
-julia_version = v"1.5.3"
-
 name = "FastJet_Julia_Wrapper"
 version = v"0.8.6"
```

Comment:
Note that if you change the `julia_compat`, you really must update the version of this JLL